po/fr.po

```diff
@@ -888,25 +884,24 @@ msgid "Save File - %s"
 msgstr "Enregistrer le fichier - %s"
 
 #: src/fileurlreader.cpp:28
-#, fuzzy, c-format
+#, c-format
 msgid "Error: failed to read URLs from file \"%s\": %s"
-msgstr "Erreur : Echec de lecture d'URLS depuis le fichier \"%s\" (%s)"
+msgstr "Erreur : échec de lecture d'URLS depuis le fichier \"%s\": %s"
```

Comment:
```suggestion
msgstr "Erreur : échec de lecture d'URLS depuis le fichier \"%s\" : %s"
```

(I don't know French, but apparently the colon is always delimited by spaces on both sides?)

Reply:
Indeed. Should have paid more attention to typographic rules instead of try to stick to much to original string.